### PR TITLE
Merge plugin-init and initInstance

### PR DESCRIPTION
**Pull request by 1000TurquoisePogs**
plugins-init.js was a replacement for plugins-init.sh to leverage quickjs configmgr.
however, this was a strange decision - we already had a JS configure step for app-server... initInstance.js

By combining these two into one, we accomplish code cleanup, decrease complexity, increase performance via reduction of processes, and resolve a bug seen by at least one user in which plugins-init.js would not work for them and they were forced to use plugins-init.sh (via ZWE_components_app_server_zowe_useConfigmgr)

Now, ZWE_components_app_server_zowe_useConfigmgr no longer has a use, and both plugins-init.sh and plugins-init.js can be removed.

This must not have a difference in behavior of plugin initialization. It is intended only to do the same actions while doing so in a cleaner and more performant way.

### CODE REVIEW

File: lib/initInstance.js [hunk 1]
```diff
@@ -42,43 +44,43 @@ const versionLocation = path.join(destination, 'component.json');
...
-fs.mkdirSync(recognizersPluginStorage, {mode: initUtils.FOLDER_MODE, recursive: true});
+initUtils.mkdirp(recognizersPluginStorage, initUtils.FOLDER_MODE);
 const actionsPluginStorage = path.join(config.instanceDir, 'ZLUX/pluginStorage/org.zowe.zlux.ng2desktop', 'actions');
-fs.mkdirSync(actionsPluginStorage, {mode: initUtils.FOLDER_MODE, recursive: true});
+initUtils.mkdirp(actionsPluginStorage, initUtils.FOLDER_MODE);
 
 const instanceConfig = path.join(config.instanceDir, 'serverConfig');
 //750 specifically, to keep server config secure
-fs.mkdirSync(instanceConfig, {mode: 0o0750});
+initUtils.mkdirp(instanceConfig, 0o0750);
```
Comment (DivergentEuropeans) [inline lib/initInstance.js:67]: This was not recursive, but made recursive. 
is OK?

Comment (DivergentEuropeans) [reply lib/initInstance.js:67]: this prob a bugfix

**Review by DivergentEuropeans — APPROVED** [review]
The changres are large but this appears to be mostly a code move. Makes sense to me, needs testing. I'll msg QA

File: lib/initInstance.js (hunk 1 above)
Comment (1000TurquoisePogs) [reply lib/initInstance.js:67]: different but harmlessly so, thanks for  spotting.

**Comment by 1000TurquoisePogs** [comment]
Merging due to completed test & review https://github.com/zowe/zowe-install-packaging/pull/3867

**Pull request merged**